### ISSUE TITLE
readbuffer 

### ISSUE THREAD

**Issue by draco37** [Fleker/CumulusTV#283]
theres anyway that you can add a setting for the read buffer?so we can  set read ahead because the iptv streams buffer a lot at least for some of us with slow internet connections.

**Comment by Fleker** [comment]
Your mean have 5 seconds or so ahead in the buffer before playing?

**Comment by draco37** [comment]
yes or let us choose even more seconds if the internet connection is too slow.
